tests/aeroval/test_modelmaps_helpers.py

```diff
@@ -1,5 +1,3 @@
-import pytest
-from ..conftest import data_tm5
```

Comment:
does this work if one is not importing `data_tm5`

Reply:
`data_tm5` is a fixture, so it is not used here. `pytest` will use the fixture when running the test, but the test_module does not need to import it.

Reply:
... and pytest would fail at the test discovery stage if an import would be missing